pybamm/expression_tree/function_parameter.py

```diff
@@ -23,5 +23,13 @@ class FunctionParameter(pybamm.UnaryOperator):
 
     """
 
-    def __init__(self, name, child):
+    def __init__(self, name, child, diff_variable=None):
```

Comment:
the docstring for this class doesn't match the `__init__`, and needs to cover `diff_variable`